.env.example

```
@@ -1,37 +1,82 @@
+# REQUIRED: secret used by FastAPI session middleware — change this before running in production
+FASTAPI_SECRET_KEY=your_super_secret_key_here
+
+# Optional: enable debug/reload when running the app directly
+# FASTAPI_DEBUG=False
+
+# -----------------------------
+# FalkorDB connection (REQUIRED / preferred)
+# -----------------------------
+# Preferred: single connection URL. Edit to point at your FalkorDB/Redis instance.
+# Example: redis://localhost:6379/0
+FALKORDB_URL=redis://localhost:6379/0  # REQUIRED - change to your FalkorDB URL
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)